packages/firestore/src/local/simple_db.ts

```diff
@@ -342,6 +356,17 @@ export class SimpleDbStore<KeyType extends IDBValidKey, ValueType> {
     return wrapRequest<void>(request);
   }
 
+  // If we ever need more of the count variants, we can add overloads. For now,
+  // all we need is to count everything in a store.
+  //
+  // Returns the number of rows in the store. Execution time, off of the event
+  // loop, is O(# rows).
```

Comment:
Use `/* ... */` style comment?  And I'm not sure what "off of the event loop" is referring to. I would probably drop it unless there's something special about this method compared to other methods on this class...

Reply:
The execution time is O(n), but the execution occurs on a different thread, so it does not block the event loop. It just means the amount of time until the you get the callback is proportional to the number of items that need to be counted. It was previously requested that I note that it was O(n). I don't feel too strongly either way...

For now I switched the comment style and reworded it slightly.

Reply:
There are no threads in JavaScript [well, excluding web workers] and everything runs on the event loop.  So if you want to say something, you could say, "Execution time is O(# rows), although asynchronous." But since it returns a PersistencePromise, it's probably not worth calling out that it's asynchronous.

Reply:
Well, yes, there are not multiple threads running js code, but the VM has multiple threads, and it appears that at least in chrome, indexeddb gets its own thread.

Regardless, I dropped the comment about running time. If anyone wants it back, they can send a PR.